common-lib/src/main/scala/com/gu/mediaservice/lib/bbc/components/BBCMetadataStore.scala

```diff
@@ -0,0 +1,55 @@
+package com.gu.mediaservice.lib.bbc.components
+
+import com.gu.mediaservice.lib.BaseStore
+import com.gu.mediaservice.lib.config.CommonConfig
+import org.joda.time.DateTime
+import play.api.Logger
+import play.api.libs.json.Json
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+
+class MetadataStore(bucket: String, config: CommonConfig)(implicit ec: ExecutionContext)
```

Comment:
Might be best for consistency to name this class `BBCMetadataStore` to match the file.